strongbox-storage/strongbox-storage-api/src/main/java/org/carlspring/strongbox/providers/repository/HostedRepositoryProvider.java

```diff
@@ -157,28 +158,26 @@ public RepositoryPath resolvePath(String storageId,
         Repository repository = getConfiguration().getStorage(storageId).getRepository(repositoryId);
         if (repository == null)
         {
-            logger.warn(String.format("Trying to resolve repository which not exists [%s].", repositoryId));
+            logger.error(String.format("Trying to resolve repository which not exists [%s].", repositoryId));
```

Comment:
Cosmetic: A slight correction in the error message to make it sound better: `"Trying to resolve repository which does not [%s]."`.

Reply:
done